admin-part/src/main/kotlin/Config.kt

```diff
@@ -18,7 +18,7 @@ package com.epam.drill.plugins.test2code
 internal class RuntimeConfig(private val pluginId: String) {
     val realtime: Boolean = sysProp("features.realtime")?.toBoolean() ?: true
 
-    val realtimeCache: Boolean = sysProp("features.realtime.cache")?.toBoolean() ?: realtime
+    val realtimeCalculationCache: Boolean = sysProp("features.realtime.cache")?.toBoolean() ?: realtime
```

Comment:
why did you rename this?

Reply:
we discussed naming with a @LikhanovMaxim  and came to the conclusion that realtimeCache sounds 
not clear enough, I didn't have time to change it in the last commit, so I'll change it in this